### ISSUE TITLE
PEAR.Functions.FunctionCallSignature.Indent behavior changed

### ISSUE THREAD

**Issue by JDGrimes** [squizlabs/PHP_CodeSniffer#721]
This is a followup to #698.

3f212a0e4278c4a5dcb2f5ed3c46c82fa507979f broke sniffing this code:

``` php
<?php
some_function(
    $arg
    , array(
        'a' => true,
        'b' => 'c',
    )
);
```

Previously, this would give no errors. However, now you get this:

```
$ phpcs --standard=PEAR --sniffs=PEAR.Functions.FunctionCallSignature test.php

FILE: test.php
----------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 2 LINES
----------------------------------------------------------------------
 5 | ERROR | [x] Multi-line function call not indented correctly;
   |       |     expected 4 spaces but found 8
 6 | ERROR | [x] Multi-line function call not indented correctly;
   |       |     expected 4 spaces but found 8
----------------------------------------------------------------------
PHPCBF CAN FIX THE 2 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------

Time: 229ms; Memory: 2Mb

```

Moving the comma from line 4 to line 3 will fix the issue. However, I prefer to put commas in front in function calls.

**Comment by gsherwood** [comment]
There should be an error thrown if the command is not the last content on the line (as per all the included standards) so you should end up with errors, but not this indent error. The fact that you got no errors before was a mistake.

I understand that you like putting commas out the front, but no included standards allow this. So you'll need to mute that error once I add it in. I'll post how to do that once it is done.

**Comment by gsherwood** [comment]
This has actually been fixed as a result of bug #820, although I still need to add the error message about the placements of commas.

**Comment by gsherwood** [comment]
Closing as I'm not going to add an error about the comma - it hasn't come up in over 2 years.
